src/components/Toast/README.md

```diff
@@ -91,23 +99,30 @@ Messages should be:
 - Product updated
 - Collection added
 - Customer updated
-- No internet connection
+- Internet disconnected
+- Connection timed out
 
 #### Don’t
 
+- No internet connection
 - Your product has been successfully updated
 - We were unable to save the customer
 - Your Order was Archived Today
 - Discount: Saved successfully
 
 <!-- end -->
 
-### Action
+### Toasts with actions
+
+Only include actions in toasts if the same action is available elsewhere on the page. For example:
+
+- If merchants need to reload a section, offer the call to action [Reload] in the toast message. If they miss the toast, they can also refresh the entire page.
+- If merchants delete an image, offer the option to [Undo] the deletion. If they miss it in the toast, they can still retrieve it from {x}.
```

Comment:
Still waiting for clarity on what `{x}` might be.

Reply:
@clauderic Do you have any input on a scenario where there would be an undo action and where that action could be taken elsewhere?

Reply:
🙏 Thank you!